### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by Aces-and-Jacks** [PingIsFun/HitBoxPlus#6]
https://mclo.gs/n6QVYzC

**Comment by PingIsFun** [comment]
```
Caused by: java.lang.RuntimeException: Non-unique Mixin config name modid.mixins.json used by the mods hitboxplus and organized_tabs
```

Looks like there is an compatibility issue with organized_tabs

**Comment by PingIsFun** [comment]
Will be fixed in the next release. I'm currently on vacation so you will have to wait a couple days for the fix.

**Comment by albrtkinn** [comment]
https://github.com/PingIsFun/HitBoxPlus/issues/6#issue-1433477583

**Comment by PingIsFun** [comment]
> https://github.com/PingIsFun/HitBoxPlus/issues/6#issue-1433477583

idk what do you mean with this

**Comment by PingIsFun** [comment]
Fixed in 0.1.3
